src/common/webview.cpp

```diff
@@ -224,6 +225,51 @@ wxString wxWebView::GetUserAgent() const
     return userAgent;
 }
 
+bool wxWebView::RunScript(const wxString& javascript, wxString* output) const
+{
+    m_syncScriptResult = -1;
+    m_syncScriptOutput.Clear();
```

Comment:
Very minor, but we prefer to use `std::string`-like methods in the new code:
```suggestion
    m_syncScriptOutput.clear();
```